### PR TITLE
Fixed smart quotes

### PR DESCRIPTION
**Pull request by svick**
Some of these changes fix code, most are just about using a uniform style.

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/servicediscovery.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Specifies the discoverability of service endpoints.
 ## Syntax  
   
 ```  
-<behaviors>  <serviceBehaviors>    <behavior name=String">      <serviceDiscovery>        <announcementEndpoints>              <endpoint name="String”                        kind="Type" />        </announcementEndpoints>        <discoveryEndpoints>              <endpoint name="String”                        kind="Type" />        </discoveryEndpoints>      </serviceDiscovery>    </behavior>  </serviceBehaviors></behaviors>  
+<behaviors>  <serviceBehaviors>    <behavior name=String">      <serviceDiscovery>        <announcementEndpoints>              <endpoint name="String"                        kind="Type" />        </announcementEndpoints>        <discoveryEndpoints>              <endpoint name="String"                        kind="Type" />        </discoveryEndpoints>      </serviceDiscovery>    </behavior>  </serviceBehaviors></behaviors>
```
Comment (svick) [inline docs/framework/configure-apps/file-schema/wcf/servicediscovery.md]: There's quite a lot of XML code blocks formatted terribly like this one.

They will need to be fixed in a separate PR.

File: docs/framework/configure-apps/file-schema/windows-identity-foundation/claimsauthenticationmanager.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Registers a claims authentication manager for the incoming claims.
   
 ```  
 \<system.identityModel>  
-    \<identityConfiguration name=”MyIdentity”>  
+    \<identityConfiguration name="MyIdentity">  
       <claimsAuthenticationManager type="MyNamespace.CustomClaimsAuthenticationManager, MyAssembly"/>          
     </identityConfiguration>  
 \</microsoft.identityModel>
```
Comment (svick) [inline docs/framework/configure-apps/file-schema/windows-identity-foundation/claimsauthenticationmanager.md]: There are also some XML code blocks like this one with incorrect escaping.

**Review by guardrex — APPROVED** [review]

**Comment by GuardRex** [comment]
@svick That's a beauty! :rocket:

Malformed XML code blocks: https://github.com/dotnet/docs/issues/2149
Escapes in code blocks: https://github.com/dotnet/docs/issues/2148
